src/NoTargets/Sdk/Sdk.targets

```diff
@@ -72,9 +72,9 @@
   <Import Project="$(CustomAfterNoTargets)" Condition="'$(CustomAfterNoTargets)' != '' and Exists('$(CustomAfterNoTargets)')" />
 
   <!-- 
-    Microsoft.Managed.Targets is imported by the managed language target files, but most of the msbuild tasks are actually in Microsoft.Common.Currentversion.targets.
+    Microsoft.Managed.Targets is imported by the managed language target files in MSBuild 16.0 and above, but most of the msbuild tasks are actually in Microsoft.Common.Currentversion.targets.
     So import it when the managed targets do not get imported.
   -->
-  <Import Project="$(MSBuildToolsPath)\Microsoft.Managed.targets" Condition="'$(ManagedLanguageTargetsGotImported)' != 'true'"/>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.Managed.targets" Condition="'$(MSBuildAssemblyVersion)' >= '16.0' And '$(ManagedLanguageTargetsGotImported)' != 'true'" />
```

Comment:
I was told in <https://github.com/MicrosoftDocs/visualstudio-docs/issues/4696#issuecomment-586433557> that MSBuildAssemblyVersion is superseded by MSBuildVersion. Is MSBuildVersion not yet available in some of the versions you want to recognize here?

Reply:
Yeah technically `MSBuildVersion` is the `Major.Minor.Build` and `MSBuildAssemblyVersion` is just `Major.0`

https://github.com/microsoft/msbuild/blob/master/src/Build/Evaluation/Evaluator.cs#L1138-L1139

`MSBuildAssemblyVersion` won't be going away and `MSBuildVersion` is more precise since Microsoft started shipping more point releases like 16.1, 16.2, etc we needed more precision.  ([I actually added](https://github.com/microsoft/msbuild/pull/2819) `MSBuildVersion`)

Technically in this case I only care about MSBuild 15 vs MSBuild 16 so `MSBuildAssemblyVersion` will work.